src/frontend/lsp/ocamlmerlin_lsp.ml

```diff
@@ -227,20 +227,17 @@ let on_request :
     return (store, lsp_locs)
 
   | Lsp.Rpc.Request.DocumentSymbol {textDocument = {uri;}} ->
-    let module DocumentSymbol = Lsp.Protocol.DocumentSymbol in
```

Comment:
btw, ocaml doesn't warn about this and I had a bug some time ago with moving things around and forgetting to move an alias — back then it resulted in an incorrect (unexpected) deriving generated.